package.json

```diff
@@ -30,13 +33,20 @@
   "directories": {
     "lib": "lib/"
   },
+  "main": "lib/index.js",
   "scripts": {
-    "build": "tsc",
-    "build:test": "cd test && ./build-tests.sh",
-    "clean": "rimraf lib",
+    "build": "jlpm build:lib && jlpm build:labextension:dev",
+    "build:prod": "jlpm clean && jlpm build:lib && jlpm build:labextension",
+    "build:labextension": "jupyter labextension build .",
+    "build:labextension:dev": "jupyter labextension build --development True .",
+    "build:lib": "tsc",
+    "clean": "jlpm clean:lib",
+    "clean:lib": "rimraf lib tsconfig.tsbuildinfo",
+    "clean:labextension": "rimraf jupyterlab-latex/labextension",
+    "clean:all": "jlpm clean:lib && jlpm clean:labextension",
     "precommit": "lint-staged",
     "prettier": "prettier --write '**/*{.ts,.tsx,.js,.jsx,.css,.json,.md}'",
-    "test": "cd test && ./run-tests.sh",
+    "install:extension": "jlpm build",
```

Comment:
perhaps `jlpm build && jupyter labextension develop . --overwrite`? Does this get used someplace?